eduNEXT/platform-plugin-ontask
# feat: add translations support

### Description
This PR adds translation support to the plugin. It only adds the translation to **Spanish (Latin America)** and **Spanish (Spain)**.

### How to Test
1. Install this plugin with the changes in this branch in your environment.
2. Change the platform language in (**Account** > **Site Preferences** > **Site Language**). Please set it to **Spanish (Latin America)** or **Spanish (Spain)**.
3. In **LMS** > **Instructor** > **OnTask** you can see the template with the corresponding translations.

### Screenshots
![image](https://github.com/user-attachments/assets/1e6577d2-6f2c-411d-b301-135349930cae)
![image](https://github.com/user-attachments/assets/ffb43e77-9f6b-4929-b83c-328bc0216b04)
